app/components/podcast-track.hbs

```diff
@@ -48,36 +48,31 @@
       </button>
     {{/if}}
   {{/if}}
-  {{#if @track.scheduledShow}}
+  <div class="podcast-info mt-1 flex flex-wrap items-center">
+    {{#if @track.scheduledShow}}
     <LinkTo @route="home.show" @model={{@track.scheduledShow}}>
       {{@track.title}}
     </LinkTo>
-  {{else}}
-    <div class="mt-1">
-      <span class="text-xl">
-        {{@track.title}}
-      </span>
-    </div>
-  {{/if}}
-  <div class="flex space-x-2">
+    {{else}}
+    <span class="text-xl">
+      {{@track.title}}
+    </span>
+    {{/if}}
     {{#each @track.labels as |label|}}
-      <a
-        class="track-label
-          text-df-yellow
-          classic:bg-df-pink
-          blm:bg-black
-          hover:text-white
-          p-1
-          uppercase
-          border-solid
-          border-white
-          border-2
-          p-x-2
-          font-extrabold"
-        {{on "click" (fn this.selectLabel label.name)}}
-        href="#">
-        {{label.name}}
-      </a>
+    <a class="track-label
+                text-df-yellow
```

Comment:
more nitpicky indentation, but i'd rather have most of this stored in the CSS if it requires this many lines. especially with the click handler hiding at the bottom makes it easy to miss